src/libponyc/ast/ast.c

```diff
@@ -541,7 +541,24 @@ void ast_resetpass(ast_t* ast)
 const char* ast_get_print(ast_t* ast)
 {
   assert(ast != NULL);
-  return token_print(ast->t);
+
+  switch(token_get_id(ast->t))
+  {
+    case TK_STRING:
+    {
```

Comment:
AFAIK you can't define a symbol in a case statement without wrapping in curly braces.